examples/logs-basic/Cargo.toml

```diff
@@ -2,6 +2,8 @@
 name = "logs-basic"
 version = "0.1.0"
 edition = "2021"
+authors = ["The OpenTelemetry Authors <cncf-opentelemetry-contributors@lists.cncf.io>"]
```

Comment:
RFC 3052 recommends against the use of `authors`, so I don't think we should be adding it here.